manifests/authoritative.pp

```diff
@@ -10,6 +14,14 @@
 
   include "powerdns::backends::${powerdns::backend}"
 
+  file { $powerdns::authoritative_config:
+    ensure => 'file',
+    owner  => 'root',
+    group  => $group,
+    mode   => '0640',
```

Comment:
This might potentially break things on other distributions, if the group is missing or it's not running under this group or `root`.

Reply:
Also: does this mean PowerDNS does not create a config anymore during installation?

Reply:
Config file will still be installed with the package installation, but managing it like that isn't going to work. It will definitely break on other distros

Reply:
Interesting. I can't speak to what the behavior is on other distros, however, the file was created with 0600 permissions on Rocky 9.5 in my testing.

I'll rework it so this change only applies on Rocky9 or perhaps Redhat 9 generally.

Reply:
I've reworked this so the group assignment only occurs on Redhat family machines.